docker/setup

```diff
@@ -38,7 +46,25 @@ puts 'Configuring environments'.yellow
     dotenv_file.puts "S3_BUCKET=europeana-stories-#{rails_env}"
     dotenv_file.puts 'S3_HOST=localhost'
     dotenv_file.puts 'S3_PATH_STYLE=1'
+    dotenv_file.puts 'REDIS_PORT=3003'
+  end
+
+  # Write the AWS bucket policy
+  policy_dst = "#{minio_dir}/.minio.sys/buckets/#{s3_bucket}"
+  policy_file_path = "#{policy_dst}/policy.json"
+  policy = JSON.parse(File.read(File.expand_path('../config/policy.json', __FILE__)))
```

Comment:
If all that lines 55-62 achieve is to insert the bucket name, I suggest instead doing:

```json
// policy.json
// ...
"Resource": ["arn:aws:s3:::%{bucket}"],
// ...
"Resource": ["arn:aws:s3:::%{bucket}/*"],
//...
```
```ruby
# docker/setup
policy = File.read(File.expand_path('../config/policy.json', __FILE__))
policy = format(policy, bucket: "europeana-stories-#{rails_env}")
```